Source/WebCore/dom/Document.cpp

```diff
@@ -2989,15 +2989,28 @@ void Document::collectRangeDataFromRegister(Vector<WeakPtr<HighlightRangeData>>&
 {
     for (auto& highlight : highlightRegister.map()) {
         for (auto& rangeData : highlight.value->rangesData()) {
-            // FIXME: For live ranges, we can optimize by only performing this when the range changed.
+            // Skip if positions are not null for static range.
```

Comment:
This comment just repeats what the code already conveys. Pease remove.
Alternatively, add "why" comment instead of "what" comment.